bin/bl-xdg-autostart

```diff
@@ -21,27 +20,39 @@
 #   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 #   GNU General Public License for more details.
 
-ME="bl-xdg-autostart"
-VERSION="1.1"
-
-import os, glob, sys
-try:
-    from xdg import BaseDirectory
-    from xdg.DesktopEntry import DesktopEntry
-    from xdg.Exceptions import ParsingError
-except ImportError:
-    print
-    print >>sys.stderr, "ERROR:", ME, "requires PyXDG to be installed"
-    print
-    sys.exit(1)
-
-def main(argv=sys.argv):
-    if "--help" in argv[1:]:
-        show_help()
-        return 0
-    if "--version" in argv[1:]:
-        show_version()
-        return 0
+from argparse import ArgumentParser, ArgumentDefaultsHelpFormatter
+import os
+import glob
+
+from xdg import BaseDirectory
+from xdg.DesktopEntry import DesktopEntry
+from xdg.Exceptions import ParsingError
+
+def main() -> int:
+    ap = ArgumentParser(
+        description="This tool will run xdg autostart .desktop files",
+        formatter_class=ArgumentDefaultsHelpFormatter,
```

Comment:
Suggest removing this line, as the argument help `(default: *)` items are confusing.

Reply:
"This line" refers to line 34 in case GitHub does not make that clear.
Removing it eliminates the (default: *) items I referred to above.

Reply:
Line 23 import of ArgumentDefaultsHelpFormatter would presumably also be unnecessary.

Reply:
I always use the DefaultsHelpFormatter but it indeed might be confusing if somebody doesn't know what the defaults: statements mean ("flag is of course false", "default environment list is of course an empty list"). You can drop it.